packages/fx-core/src/common/tools.ts

```diff
@@ -51,11 +51,11 @@ import { CloudResource } from "@microsoft/teamsfx-api/build/v3";
 
 Handlebars.registerHelper("contains", (value, array, options) => {
   array = array instanceof Array ? array : [array];
-  return array.indexOf(value) > -1 ? options.fn(this) : "";
+  return array.indexOf(value) > -1 ? this : "";
```

Comment:
Nit: remove `options` from argument list? since it's useless now.

Reply:
thanks!
updated